eux.conf

```diff
@@ -0,0 +1,20 @@
+server {
+        listen       80;
+        server_name  0.0.0.0;
+        #root         /usr/share/nginx/html/eux;
+
+        location / {
+        	root html;
+        	index index.html;
+        }
+
+		location /api {
+          proxy_pass https://eux.nais.preprod.local/api;
+        	#proxy_pass http://localhost:3002/eux-api;
+        	#-- nginx lokalt
+        	#proxy_pass http://localhost:8000/eux/api;
+        	#-- for NAIS miljo ???
```

Comment:
Adressene under er for spesifikke servere innenfor NAV. 
Den først ikke utkommenterte adressen er for NAIS miljøet.

Reply:
Ok, har forsøkt å rydde opp i gruppering og kommentarer.